scripts/lc-builds/toss3_hipcc.sh

```diff
@@ -50,11 +62,25 @@ cmake \
   -DHIP_CLANG_PATH=/opt/rocm-${COMP_VER}/llvm/bin \
   -DCMAKE_C_COMPILER=/opt/rocm-${COMP_VER}/llvm/bin/clang \
   -DCMAKE_CXX_COMPILER=/opt/rocm-${COMP_VER}/llvm/bin/clang++ \
-  -DHIP_HIPCC_FLAGS=--offload-arch=${COMP_ARCH} \
-  -C ../host-configs/lc-builds/toss3/hip_link_X.cmake \
+  -DHIP_CLANG_FLAGS="${HIP_CLANG_FLAGS}" \
+  -C "../host-configs/lc-builds/toss3/${HOSTCONFIG}.cmake" \
```

Comment:
Should we also duplicate this file for TOSS4?

Reply:
I recall that the script worked on TOSS3 (corona before the upgrade) and TOSS4 (rznevada). So we may be able to get away with only one. Although it may not be a bad idea to duplicate for now while we have TOSS3 and TOSS4 systems on the floor.

Reply:
I think this probably should have just been toss4 from the beginning.

Reply:
Let's duplicate them then.